README.md

```diff
@@ -231,6 +245,31 @@ xhr.XMLHttpRequest = MockXMLHttpRequest
 xhr.XDomainRequest = MockXDomainRequest
 ```
 
+## Query string support
+There are many ways to stringify query parameters; consequently, `xhr` makes no
+assumptions about how to handle them, and does not support the `qs` option out
+of the box.
+
+To support the `qs` option, define an `xhr.qsSerialize` function. This function
+accepts the value of `options.qs` as its first argument, and returns a string
+that is appended to the URL.
+
+You do not need to include a leading "?" in the value that you return from
+`xhr.qsSerialize`.
+
+```js
+var xhr = require('xhr')
+var qs = require('qs')
+
+xhr.qsSerialize = qs.stringify
```

Comment:
^ this line is very convincing towards naming it qsStringify after all. 
I'll keep thinking about it after I merge this PR into v3 branch